res/qml/images/jump_backward.svg

```diff
@@ -0,0 +1 @@
+../../skins/LateNight/palemoon/buttons/btn__beatjump_left.svg
```

Comment:
This is still a malformed SVG

Reply:
This is a symlink so either `../../skins/LateNight/palemoon/buttons/btn__beatjump_left.svg` is malformed (and thus the problem is a in `main`) or your parser doesn't support those.

Reply:
Ah so this is a Windows issue. It looks like on Windows Symlinks become plain text files just with the symlink target as text. That's why Firefox complains and Inkscape refused to open the file.